src/dreyfus_index.erl

```diff
@@ -157,9 +157,27 @@ handle_call(info, _From, State) -> % obsolete
     Reply = info_int(State#state.index_pid),
     {reply, Reply, State}.
 
+handle_cast({ddoc_updated, DDocResult},
+            #state{
```

Comment:
on principle, we only extract terms in the function head if we are matching on them. so pull Sig and DbName out on a separate line. This will also make the function head much smaller and it will fit on one line.

```
handle_cast({ddoc_updated, DDocResult}, #state{} = State) ->
    #index{sig = Sig, dbname = DbName} = State#state.index,
```